common-tools/clas-detector/src/main/java/org/jlab/detector/pulse/Pulse.java

```diff
@@ -0,0 +1,46 @@
+package org.jlab.detector.pulse;
+
+import org.jlab.detector.base.DetectorDescriptor;
+
+/**
+ * Just a dumb data container
+ */
+public class Pulse {
+
+    public DetectorDescriptor descriptor;
+    public long timestamp;
+    public float integral;
+    public float time;
+    public float pedestal;
+    public long flags;
+    public int id;
+
+    public float adcMax;
+    public float timeRiseCFA;
```

Comment:
Since this class is for general use, members' names should be more generic removing the CFA or CFD suffix.

Also the pulse "rise" and "fall" are usually referred to as leading and trailing edge that could be better names